viscy/data/cell_division_triplet.py

```diff
@@ -0,0 +1,412 @@
+import logging
+import random
+from pathlib import Path
+from typing import Literal, Sequence
+
+import numpy as np
+import torch
+from monai.transforms import Compose, MapTransform
+from torch import Tensor
+from torch.utils.data import Dataset
+
+from viscy.data.hcs import HCSDataModule
+from viscy.data.triplet import (
+    _transform_channel_wise,
+)
+from viscy.data.typing import DictTransform, TripletSample
+
+_logger = logging.getLogger("lightning.pytorch")
+
+
+class CellDivisionTripletDataset(Dataset):
+    # Hardcoded channel mapping for .npy files
```

Comment:
Where is this used?

Reply:
this is for the RPE dataset

Reply:
But where is this channel mapping attribute used? I can't find it in the code base.